New chart for function-controller (#6948)
kyma-project/kyma: resources/function-controller/README.md

## Prerequisites

> - Kubernetes Cluster (v1.15.3)

Reviewer: ```suggestion
- Kubernetes cluster (v1.15.3)
```

| Variable        | Description | Sample value | 
| --------------- | ----------- | --------|
| **FN_REGISTRY**   | The URL of the container registry Function images will be pushed to. Used for authentication.  | `https://gcr.io/` for GCR, `https://index.docker.io/v2/` for Docker Hub|
| **FN_REPOSITORY** | The name of the container repository Function images will be pushed to. | `gcr.io/my-project` for GCR, `my-user` for Docker Hub |

> 2. Apply crds:

Reviewer: ```suggestion
2. Apply CRDs:
```

    ```bash
    kubectl apply -f dev/crds
    ```

3. Install the Function Controller charts:

    ```bash
    NAME=function-controller

>     NAMESPACE=serverless-system

Reviewer 2: ATM chart readme and project readme are inconsistent as it comes to namespace in which config maps containing Dockerfiles are created. Since those config maps must be in the same namespace in which the function may be created, it would be great to align it or at least document it to avoid confusion.

                 --set global.ingress.domainName="${DOMAIN_NAME}" \
                 --tls
    ```

### Run the first function

> Currently, there is no UI support for the new Function Controller. Follow [these](https://github.com/kyma-project/kyma/blob/master/components/function-controller/README.md#create-a-sample-hello-world-function) instructions to deploy function in `serverless-system

Reviewer: ```suggestion
Currently, there is no UI support for the Function Controller. Follow [these](https://github.com/kyma-project/kyma/blob/master/components/function-controller/README.md#create-a-sample-hello-world-function) instructions to deploy a function in the Namespace you specified during the chart installation (step 3).
```

Reviewer: Is `specify` ok for you? `determine` is more "to establish exactly by research or calculation".

Author: It's ok! Thanks

> ` namespace determinate in the previous step.

Reviewer: ```suggestion
```
